docs/playbooks/Path of the Writer Episode 1 0 617ae65ca624491ab516f40c8414b5ab.md

```diff
@@ -1,4 +1,9 @@
-# Path of the Writer Episode 1.0
+---
+title: 📚 Path of the Writer Episode 1.0
```

Comment:
Thanks for adding this in @pvblick-vniversal-friend. 

We do need to fix the filename and add it to the `sidebar.js` file which is the classic wiki update gotcha! Not sure what happened with the file name but I expect the spaces will b0rk Docusaurus and we don't need the ` 0 617ae65ca624491ab516f40c8414b5ab` there. If you're not sure how to do that, I'm happy to help you do it or I can sort it in a jiffy.

Reply:
I'll try my hand at it and then I'll leave it to you to review it.

Reply:
@luxumbra done!